### ISSUE TITLE
Request: chpldoc support non-block comment style

### ISSUE THREAD

**Issue by BryantLam** [chapel-lang/chapel#14207]
Feature request. For `chpldoc`, support non-block comment styles. I personally avoid using block/multi-line comments whenever possible.

```terminal
$ chpldoc Library.chpl --comment-style='///'
error: comment label should start with /*
```
`chpldoc version 1.21.0 pre-release (b736500a)`

**Comment by bradcray** [comment]
Just to make sure, is your thought that all adjacent comment lines with matching format immediately preceding the declaration up through the previous non-comment or blank line would be the documentation?  That is:

```chapel
/// This is not part of x's documentation

/// This is part of x's documentation
/// as is this
var x: int;
```

**Comment by BryantLam** [comment]
Yes, I'd agree with that interpretation; adjacent comment lines of the same style form a single document comment. I would look up how Doxygen would interpret this code simply to note if there are any corner cases that need to be handled.

**Comment by lydia-duncan** [comment]
Oh, fun!  I think that's do-able but it will definitely require some special casing at parse time . . .
